code/modules/mob/living/simple_animal/abnormality/zayin/wellcheers.dm

```diff
@@ -92,3 +95,32 @@
 	H.adjustBruteLoss(rand(-2, 3) * REAGENTS_EFFECT_MULTIPLIER)
 	H.adjustSanityLoss(rand(-2, 3) * REAGENTS_EFFECT_MULTIPLIER)
 	return ..()
+
+//Shrimple boat stuff
+/turf/open/water/deep/saltwater/extradeep
+
+/turf/open/water/deep/saltwater/extradeep/Entered(atom/movable/thing, atom/oldLoc) //Drowning code - you won't make it back to the station alive.
+	if(!target_turf || is_type_in_typecache(thing, forbidden_types) || (thing.throwing && !istype(thing, /obj/item/food/fish || /obj/item/aquarium_prop )) || (thing.movement_type & (FLOATING|FLYING))) //replace this with a varient of chasm component sometime.
+		return ..()
+	if(isliving(thing))
+		var/mob/living/L = thing
+		if(L.movement_type & FLYING)
+			return ..()
+		if(!ishuman(L))
+			qdel(L)
+			return
+		var/mob/living/carbon/human/H = L
+		INVOKE_ASYNC(src, .proc/Drown, H)
+
+/turf/open/water/deep/saltwater/extradeep/proc/Drown(mob/living/carbon/human/H)
+	H.Stun(30 SECONDS)
+	H.visible_message("<span class='userdanger'>[H] falls in the water and starts to squirm frantically! It looks like they're going to drown!</span>", "<span class='userdanger'>The sea is far too dangerous! You slip into the depths...</span>")
+	playsound(src, 'sound/voice/human/wilhelm_scream.ogg', 50, TRUE, -3)
+	animate(H, alpha = 0,pixel_x = 0, pixel_z = 0, time = 3 SECONDS)
+	QDEL_IN(H, 3.5 SECONDS)
+	sleep(3 SECONDS)
+	playsound(src, 'sound/abnormalities/dreamingcurrent/dead.ogg', 80, TRUE, -3)
```

Comment:
This water should search a mob's contents and force move any fishing rods and nets back to the ship, probably the landmarker, otherwise it'll be stripped bare very quickly.

Reply:
Hmm, I didn't think about that. I'll look at doing that.